src/feature/featurelistcallback.h

```diff
@@ -174,4 +174,12 @@ bool FeatureCallback_webPurchase() {
 #endif
 }
 
+bool FeatureCallback_hasBalrog() {
+#if defined(MVPN_BALROG)
```

Comment:
Should this also be restricted to `FeatureCallback_inStaging()` for now?

Reply:
I think it's fine to leave this enabled for prod as well, that gives us the ability to test updates and key switching in prod too.